sdks/python/apache_beam/ml/transforms/tft_test.py

```diff
@@ -391,5 +392,81 @@ def equals_fn(a, b):
       assert_that(actual_output, equal_to(expected_output, equals_fn=equals_fn))
 
 
+class ScaleToMinMaxTest(unittest.TestCase):
+  def setUp(self) -> None:
+    self.artifact_location = tempfile.mkdtemp()
+
+  def tearDown(self):
+    shutil.rmtree(self.artifact_location)
+
+  def test_scale_to_min_max(self):
+    data = [{
+        'x': 4,
+    }, {
+        'x': 4,
+    }, {
+        'x': 4,
+    }]
```

Comment:
Nit: could we add a test where they're not all the same value?

Reply:
Yes, I added this test case, which was used in TFT tests.